sos/plugins/openstack_aodh.py

```diff
@@ -71,9 +85,10 @@ def postproc(self):
         )
 
         self.do_file_sub(
-            "/etc/aodh/aodh.conf",
-            r"(auth_url[\t\ ]*=[\t\ ]*)(.+)",
-            r"\1********",
+            "/var/lib/config-data/puppet-generated/aodh/etc/aodh/aodh.conf",
```

Comment:
Since we have the same path prefix here (although it does not force a linebreak) we could use e.g. `self.var_puppet_generated` or similar.